.circleci/config.yml

```diff
@@ -277,10 +277,10 @@ jobs:
           name: Build Docker local image
           command: |
             echo "Building Docker image: local"
-            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:local .
```

Comment:
Hi @ruirochawork , I am not sure why you want too replace the variable names with hardcoded values. But we have a standard for circleCI config file across all the mojaloop repositories

Reply:
Hi @vijayg10 , I was just testing out on why the docker build is failing. Seems like it might be an issue with population some kind of environment variable, hence why I changed the state of the PR to draft until I figure out what it is.

https://app.circleci.com/pipelines/github/mojaloop/ml-testing-toolkit-ui/673/workflows/51dd6c16-011d-4249-b696-dbe35a2c1957/jobs/2784

```
#!/bin/bash -eo pipefail
echo "Building Docker image: local"
docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:local .

Building Docker image: local
invalid argument "/ml-testing-toolkit-ui:local" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.

Exited with code exit status 125
CircleCI received exit code 125
```

Do you happen to know what the problem might be? Would appreciate if you or someone else could help me with this issue.